spec/hosts/nodes/konkong01.lsst.org_spec.rb

```diff
@@ -0,0 +1,108 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'konkong01.ls.lsst.org', :site do
+  alma8 = FacterDB.get_facts({ operatingsystem: 'AlmaLinux', operatingsystemmajrelease: '8' }).first
+  # rubocop:disable Naming/VariableNumber
+  { 'almalinux-8-x86_64': alma8 }.each do |os, facts|
+    # rubocop:enable Naming/VariableNumber
+    context "on #{os}" do
+      let(:facts) { override_facts(facts, fqdn: 'konkong01.ls.lsst.org') }
+      let(:node_params) do
+        {
+          role: 'rke',
+          site: 'ls',
+          cluster: 'konkong',
+        }
+      end
+      let(:vlan_id) { 2505 }
+      let(:rt_id) { vlan_id }
+
+      include_context 'with nm interface'
+
+      it { is_expected.to compile.with_all_deps }
+
+      it do
+        is_expected.to contain_class('profile::core::sysctl::rp_filter').with_enable(false)
+      end
+
+      it do
+        is_expected.to contain_class('clustershell').with(
+          groupmembers: {
+            'konkong' => {
+              'group' => 'konkong',
+              'member' => 'konkong[01-03]',
```

Comment:
I would suggest do from 1-5 now, do a short branch just for that would be more pain in the future...